docs/install.md

```diff
@@ -311,7 +311,10 @@ Adjust the "certbot" command below according to what domain names you want to re
 ## Running the Server
 
 - The "run.sh" script in the root of the repository will build and run the server.
-  - If you are on Windows, you should run this script from a Git Bash window.
+  - If you are on Windows, you should run this script from a Git Bash window by running `./run.sh.` If this is your first time running the server or you have changed some lines of code you have to recompile the program for it to load properly using the below 3 lines instead.
+  - `npm i`
+  - `cd packages/client/ ; ./build_client.sh ; ./esbuild_dev.sh` wait until the window ends with "waiting for changes..." before killing with ctrl-c
+  - `cd ../../ ; ./server/build_server.sh ; ./run.sh`
```

Comment:
These lines could probably use some clarification or cleanup; @Zamiell probably knows which of these are extraneous, as I only came to these by trial and error and posted them on the discord.